curated_transformers/models/hf_util.py

```diff
@@ -33,15 +49,25 @@ def convert_hf_pretrained_model_parameters(
         "xlm-roberta": _convert_roberta_base_state,
     }
 
-    return converters[hf_model.config.model_type](hf_model)
+    return converters[hf_model.config.model_type](hf_model.state_dict())
+
+
+def _rename_old_hf_names(
+    params: OrderedDict[str, torch.Tensor],
+) -> OrderedDict[str, torch.Tensor]:
+    out = OrderedDict()
+    for name, parameter in params.items():
+        name = re.sub(r"\.gamma$", ".weight", name)
+        name = re.sub(r"\.beta$", ".bias", name)
+        out[name] = parameter
+    return out
 
 
 def _convert_albert_base_state(
-    hf_model: "transformers.PreTrainedModel",
+    params: OrderedDict[str, torch.Tensor],
 ) -> Dict[str, torch.Tensor]:
-    out = {}
-
-    state_dict = dict(hf_model.state_dict().items())
+    # Strip the `bert` prefix from XLM-Roberta model parameters.
```

Comment:
This comment seems incorrect? `bert` should be `albert` according to the substitution below, and I think you mean "`Albert` model parameters"?

Reply:
Fixed both comments.